File: desktop/ShareCLITray/Sources/ShareCLICore/TrayPoll.swift
/// Periodic tray poll cadence (FR-007 / AC-007.53).
///
/// MUST match `sharecli_tray_linux::poll::TRAY_POLL_INTERVAL_SECS` and
/// `sharecli_tray_windows::poll::TRAY_POLL_INTERVAL_SECS` (Linux/Windows parity).
public enum TrayPoll {
    public static let intervalSeconds: UInt64 = 3
    public static let intervalNanoseconds: UInt64 = intervalSeconds * 1_000_000_000
}

File: windows/ShareCLITray/TrayPoll.cs
namespace ShareCLITray;

/// Periodic tray poll cadence (FR-007 / AC-007.52).
/// MUST match `sharecli_tray_windows::poll::TRAY_POLL_INTERVAL_SECS` (Linux/Swift parity).
public static class TrayPoll
{
    public const int IntervalSeconds = 3;
}
